app/controllers/catalog_controller.rb

```diff
@@ -251,16 +251,18 @@ class CatalogController < ApplicationController
     # config.add_show_field 'subtitle_vern_display', :label => 'Subtitle'
 
     # Top fields in show page / prioritized information
-    config.add_show_field 'author_display', label: "Author/\u200BArtist", browse_link: :name, if: false
+    # Top fields must include `if: false`, or they will be duplicated in the Details section
```

Comment:
Not a blocker, but it does seem awkward to have to remember to use the `if: false` as well as the `*_top_field: true`.  It would be more ergonomic to just need one.  Maybe we could open another ticket for that as future work?

Reply:
Oooh, also noting that if we get rid of the `if: false` requirement in the CatalogController, we could scrap the `AlwaysShowMetadataFieldComponent` too.  Literally its only job is to make those top_fields show despite being set to `if: false`.

Reply:
Yes, I was wanting to get rid of the `if: false`, but then was worrying I was getting into perfectionism, and thought for now I'd get it to where we can get in the 583 changes, then keep refactoring. I forgot about the AlwaysShowMetadataFieldComponent class, that would be nice to be able to remove.